File: programmers/LV1_2016년.cpp
#include <bits/stdc++.h>
using namespace std;

string solution(int a, int b) {
    string answer = "";
    vector<string> day = { "FRI", "SAT", "SUN", "MON", "TUE", "WED", "THU"};
    vector<int> dayCnt = {31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31};
    int days = 0;
    
    for(int i=0; i<a-1; i++) {
        days += dayCnt[i];
    }

    days += b-1;
    answer = day[days%7];
    
    return answer;
}

File: _python/programmers/LV1_2016년.py
def solution(month, day):
    answer = ''
    
    day_word = ['FRI', 'SAT', 'SUN', 'MON', 'TUE', 'WED', 'THU']
    days = [0, 31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
    
    total = 0
    
    for i in range(month):
        total += days[i]
    
    total += day -1
    total %= 7
    print(total)
    answer = day_word[total]
    
    return answer